android/src/main/java/com/frontegg/android/embedded/FronteggWebClient.kt

```diff
@@ -303,6 +303,15 @@ class FronteggWebClient(val context: Context, val passkeyWebListener: PasskeyWeb
                 return true
             }
 
+            if (url.scheme.equals(storage.deepLinkScheme, ignoreCase = true)) {
+
+                val intent = Intent(Intent.ACTION_VIEW, url)
+                intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK
+                context.startActivity(intent)
+
+                return true
```

Comment:
What is the purpose of the FLAG_ACTIVITY_NEW_TASK flag? 
When navigating to this deep link, does it close the current login activity?